sirepo/template/template_common.py

```diff
@@ -207,7 +207,10 @@ def validate_model(model_data, model_schema, enum_info):
             raise Exception('no value for field "{}" and no default value in schema'.format(k))
         if field_type in enum_info:
             if str(value) not in enum_info[field_type]:
-                raise Exception('invalid enum value: {} for {}'.format(value, k))
+                # Check a comma-delimited string against the enumeration
+                for item in re.split(",\s*", str(value)):
```

Comment:
When splitting, you want r'\s*,\s*': the "r" ensures no interpretation of the \, the single quotes are the string form we use by default, the \s* starting is looser.